tools/docker/generate_dockerfiles.py

```diff
@@ -401,11 +401,8 @@ def install_cp2k_cmake() -> str:
 WORKDIR ./build
 RUN /bin/bash -c " \
     echo 'Compiling cp2k...' && \
-    ls /opt/cp2k-toolchain/install/scalapack-2.2.1 && \
-    ls /opt/cp2k-toolchain/install/fftw-3.3.10/lib && \
     source /opt/cp2k-toolchain/install/setup && \
-    export PKG_CONFIG_PATH=/opt/cp2k-toolchain/install/libxsmm-1.17/lib:/opt/cp2k-toolchain/install/openblas-0.3.21/lib/pkgconfig:/opt/cp2k-toolchain/install/libxc-6.0.0/lib/pkgconfig:/opt/cp2k-toolchain/install/fftw-3.3.10/lib/pkgconfig:/opt/cp2k-toolchain/install/libint-v2.6.0-cp2k-lmax-5/lib/pkgconfig:/opt/cp2k-toolchain/install/plumed-2.8.0/lib/pkgconfig:/opt/cp2k-toolchain/install/superlu_dist-6.1.0/lib/pkgconfig && \
-    cmake -DCP2K_USE_COSMA=OFF -DCP2K_USE_LIBXSMM=NO -DSCALAPACK_ROOT=/opt/cp2k-toolchain/install/scalapack-2.2.1 -DCP2K_BLAS_VENDOR=OpenBLAS -DLibXC_ROOT=/opt/cp2k-toolchain/install/libxc-6.0.0 -DLibint2_ROOT=/opt/cp2k-toolchain/install/libint-v2.6.0-cp2k-lmax-5 -DDBCSR_ROOT=/opt/cp2k-toolchain/install/DBCSR-2.4.1 -DCP2K_USE_SPGLIB=ON -DCP2K_USE_LIBINT2=NO -DCP2K_USE_LIBXC=ON -DLibSPG_ROOT=/opt/cp2k-toolchain/install/spglib-1.16.2 .. && \
+    cmake -DCP2K_USE_VORI=ON -DCP2K_USE_COSMA=NO -DCP2K_USE_LIBXSMM=ON -DCP2K_BLAS_VENDOR=OpenBLAS -DCP2K_USE_SPGLIB=ON -DCP2K_USE_LIBINT2=ON -DCP2K_USE_LIBXC=ON .. && \
```

Comment:
Great work! This cmake command line looks so nice now :-)

Reply:
thanks. It is a coordinated effort. 

If you spot something fishy let us know because I had to modify all files from the toolchain to properly setup all variables. 
Next step is the spack package and it is becoming urgent as spack people are starting to update it. This PR was actually important because if I am a bit more confident now that it will work with spack. I got it working for the cpu side for now. But work in progress.

- we can also turn on GPU support.